src/hdf5plugin/__init__.py

```diff
@@ -390,9 +390,33 @@ class Zstd(_FilterRefClass):
             data=numpy.arange(100),
             **hdf5plugin.Zstd())
         f.close()
+
+    :param int clevel: Compression level. Negative values allow for faster
+        speeds at the cost of compression. Ultra compression extends from
+        20 through 22. Default: 3.
+
+    .. code-block:: python
+
+        f = h5py.File('test.h5', 'w')
+        f.create_dataset(
+            'zstd',
+            data=numpy.arange(100),
+            **hdf5plugin.Zstd(clevel=22))
+        f.close()
     """
     filter_id = ZSTD_ID
 
+    def __init__(self, clevel=None):
+        """
+        `clevel` is chosen to be consistent with the Blosc plugin.
+        The plugin C code refers to clevel as `aggression`.
+        Zstd refers to `CLevel` (e.g. `ZSTD_defaultCLevel()`)
+        and `compressionLevel` for the `clevel` argument.
+        """
+        if clevel is not None:
+            clevel = _ctypes.c_uint(clevel).value
```

Comment:
This is not need for now since negative values are clipped, but it can remain as it is if you plan on updating the HDF5Plugin-Zstandard to enable negative values.

Reply:
Resolved in [9613778](https://github.com/silx-kit/hdf5plugin/pull/164/commits/9613778c1275aab4ce282ad070b7a4e73f75e675)